### ISSUE TITLE
Reblog form: Form is missing label element

### ISSUE THREAD

**Issue by SiobhyB** [Automattic/jetpack#6975]
#### Steps to reproduce the issue

1. Enable the Reblog button on any WordPress.com or Jetpack-connected site.

2. View the button on any page of the site. When you click on it, it will reveal a text box and look similar to the following screenshot:

<img width="549" alt="screen shot 2017-04-13 at 22 01 23" src="https://cloud.githubusercontent.com/assets/2998162/25024197/e189c0ea-2094-11e7-9a32-cdcba6eb75ca.png">

3. Use [the WAVE accessibility tool](http://wave.webaim.org/) to review the site for errors.

#### Results

The tool reveals that the Reblog button is missing a label element:

<img width="431" alt="screen shot 2017-04-13 at 21 59 12" src="https://cloud.githubusercontent.com/assets/2998162/25024089/7e2b1bc0-2094-11e7-8981-20bcf0b75f79.png">

Reported in p4vt7e-3a-p2.

**Comment by jeherve** [comment]
Reblogs aren't included in Jetpack, so that's not something we'll be able to fix it here. I'd recommend opening a wpcom Trac ticket about this.

**Comment by SiobhyB** [comment]
*Face palm.* 

Thank you for pointing that out @jeherve! I've moved this to 7870-wpcom.
